whisper/whisperv5/filter.go

```diff
@@ -176,7 +183,7 @@ func (f *Filter) MatchTopic(topic TopicType) bool {
 	return false
 }
 
-func isPubKeyEqual(a, b *ecdsa.PublicKey) bool {
+func IsPubKeyEqual(a, b *ecdsa.PublicKey) bool {
```

Comment:
I'll move it to package crypto later. The exported API surface of package whisper should be about whisper and shouldn't provide generic crypto helpers.